mgmtsystem_action/data/email_template.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Email template for Action Reminders -->
+    <data noupdate="1">
+        <record id="email_template_new_action_reminder" model="mail.template">
+            <field name="name">Notify the person in charge of the new action</field>
+            <field name="email_from">${object.create_uid.company_id.email or 'noreply@localhost.com'}</field>
+            <field name="subject">${object.name}</field>
+            <field name="email_to">${object.user_id.email}</field>
+            <field name="model_id" ref="model_mgmtsystem_action"/>
+            <field name="auto_delete" eval="True"/>
+            <field name="lang">${object.create_uid.partner_id.lang}</field>
+            <field name="body_html">
+                <![CDATA[
+<p>Hello,</p>
+<p>A new action has been assigned to you : <a href="${object.get_action_url()}">${object.get_action_url()}</a></p>
+<p>${object.description}</p>
+
+<p>
+  Thank you<br />
+  --<br />
+  Odoo
+</p>]]>
+            </field>
```

Comment:
Add an Item to the Roadmap: the custom emails should be replaced by Mail Tracking features and Subtypes (like in Project Tasks and Project Issues).